CustomLookup/CustomEntitySearch.cs

```diff
@@ -0,0 +1,187 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System.Net.Http.Formatting;
+using System.Text;
+using Microsoft.AspNetCore.Routing.Template;
+
+namespace CustomLookup
```

Comment:
You can look at other skills to see how to structure your namespace. It should probably be `AzureCognitiveSearch.PowerSkills.Text.CustomEntitySearch`